group_vars/moc/vlans.yaml

```diff
@@ -0,0 +1,940 @@
+vlans:
+  10:
+    name: "CSAIL-MAIN"
+    description: "Various Openshift clusters"
+  105:
+    name: "MOC-BU-PUBLIC"
+    description: "Public IPs from BU"
+  127:
+    name: "MOC-NEU-PUBLIC"
+    description: "NEU Public IP for infrastructure"
+  204:
+    name: "MOC-GENERAL"
+    description: "Intranet (routable to internet). SNMP for client nodes."
```

Comment:
VLAN 205 should also be here https://github.com/CCI-MOC/ansible-switches/blob/main/group_vars/all/vlans.yaml#L33